code/datums/religions.dm

```diff
@@ -1492,26 +1531,174 @@
 
 /datum/religion/egypt
 	name = "Kemetism"
-	deity_names = list("Osiris", "Anubis", "Ra", "Horus", "Kek")
+	deity_names = list("Osiris", "Anubis", "Ra", "Horus", "Kek", "Thoth")
 	bible_name = "The Pyramid Texts"
 	bible_type = /obj/item/weapon/storage/bible/booze
 	male_adept = "Hem-Netjer"
 	female_adept = "Hemet-Netjer"
-	keys = list("kemetism", "egypt", "egyptian")
+	keys = list("kemetism", "egypt", "egyptian", "kangz", "we wuz kangz", "hotep", "shadilay")
 
 /datum/religion/egypt/equip_chaplain(var/mob/living/carbon/human/H)
 	H.equip_or_collect(new /obj/item/clothing/head/pharaoh(H), slot_head)
 
 /datum/religion/anprim
 	name = "Primitivism"
-	deity_name = "Grug"
+	deity_names = ("Grug","Ted Kaczynski")
 	bible_name = "Industrial Society and Its Future"
 	bible_type = /obj/item/weapon/storage/bible/booze
 	male_adept = "Primitive"
 	female_adept = "Primitive"
-	keys = list("anarcho-primitivism", "primitivism", "anprim", "primitive", "grug")
+	keys = list("anarcho-primitivism", "primitivism", "anprim", "primitive", "grug", "unabomber")
 	bookstyle = "Scrapbook"
 
 /datum/religion/anprim/equip_chaplain(var/mob/living/carbon/human/H)
 	H.equip_or_collect(new /obj/item/clothing/under/shorts/black, slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/suit/unathi/mantle(H), slot_wear_suit)
+
+/datum/religion/cornholio
+	name = "Cornholism"
+	deity_name = "The Great Cornholio"
+	bible_name = "Teachings of the Bunghole"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Cornholio"
+	female_adept = "Cornholio"
+	preferred_incense = /obj/item/weapon/storage/fancy/incensebox/novaflowers
+	keys = list("cornholio", "tp", "toilet paper", "bunghole", "beavis")
+
+//TODO: some conversion ceremony with surgical drills to make stigmatas from lake titticaca
+/datum/religion/cornholio/equip_chaplain(var/mob/living/carbon/human/H)
+	H.reagents.add_reagent(COFFEE,10) // settle down, beavis
+	H.reagents.add_reagent(SUGAR,10)
+
+/datum/religion/schizo
+	name = "Schizophrenia"
+	deity_name = "The voices inside your head"
+	bible_name = "The Madman's Ramblings"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Schizo"
+	female_adept = "Schizo"
+	preferred_incense = /obj/item/weapon/storage/fancy/incensebox/moonflowers
+	keys = list("schizophrenia", "schizo", "meds", "ss13g", "the thread", "wesley willis", "time cube", "timecube", "lifeweb")
+	bookstyle = "Scrapbook"
+
+/datum/religion/schizo/equip_chaplain(var/mob/living/carbon/human/H)
+	H.reagents.add_reagent(MINDBREAKER,10)
+	H.reagents.add_reagent(SPACE_DRUGS,10)
+
+/datum/religion/pomf
+	name = "Pomfism"
+	deity_name = "Pomf"
+	bible_name = "The Book of Pomf"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Chicken Farmer"
+	female_adept = "Chicken Farmer"
+	keys = list("pomf", "pomf123", "chicken", "chikun", "pomf chicken", "d3athrow", "badclown", "/vg/station", "/vg/station13", "pomfcoin")
```

Comment:
so are some of these. like, ckeys? actually just dropping the server name? cmon

Reply:
oh no to those either then